recipes/PyAV/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "av" %}
```

Comment:
The directory and package name don't match. So this has failed to convert. Should it be `av` or `pyav`?